fury/shaders/visible_range.glsl

```diff
@@ -0,0 +1,7 @@
+bool inVisibleRange(vec3 center)
+{
+    bool xVal = lowRanges.x <= center.x && center.x <= highRanges.x;
+    bool yVal = lowRanges.y <= center.y && center.y <= highRanges.y;
+    bool zVal = lowRanges.z <= center.z && center.z <= highRanges.z;
+    return xVal && yVal && zVal;
+}
```

Comment:
This file should be in a folder determined by its function. Perhaps, `fury/shaders/interaction`, but I'll leave the name up to you.